### PR TITLE
Add Migrations and Seeds for ExperimentAttribute and ExperimentValue

### PR DESCRIPTION
**Pull request by FinnKeller**

### CODE REVIEW

**Comment by FinnKeller** [comment]
I dont know what I did but apparently this pull request is now for issue 38 and issue 80.

File: server/database/migrations/experiment/00_Attribute.js
Comment (niklhut) [inline server/database/migrations/experiment/00_Attribute.js:1]: Please move the file top level to migrations and name it like the model, so 00_experiment_attribute

File: server/database/migrations/experiment/00_Attribute.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { DataTypes } from "sequelize"
+/**
+ * run the Attribute
+ */
+async function up({ context: queryInterface }) {
+  await queryInterface.createTable("Attribute", {
```
Comment (niklhut) [inline server/database/migrations/experiment/00_Attribute.js:6]: The name should be the model name but pluralized so `ExperimentAttributes`

File: server/database/migrations/experiment/00_Attribute.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      allowNull: false,
+      defaultValue: DataTypes.NOW,
+    },
+  })
+}
+/**
+ * revert the Experiment Attribute Migration
+ */
+async function down({ context: queryInterface }) {
+  await queryInterface.dropTable("Attribute")
```
Comment (niklhut) [inline server/database/migrations/experiment/00_Attribute.js:32]: Also adjust name here

File: server/database/migrations/experiment/00_Value.js
Comment (niklhut) [inline server/database/migrations/experiment/00_Value.js:1]: Please move the file top level to migrations and name it like the model, so 00_experiment_attribute_value

File: server/database/migrations/experiment/00_Value.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import { DataTypes } from "sequelize"
+import ExperimentAttribute from "../../models/ExperimentAttribute"
+/**
+ * run the value migration
+ */
+async function up({ context: queryInterface }) {
+  await queryInterface.createTable("Value", {
```
Comment (niklhut) [inline server/database/migrations/experiment/00_Value.js:7]: The name should be the model but in plural, so `ExperimentAttributeValues`

File: server/database/migrations/experiment/00_Value.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      allowNull: false,
+      defaultValue: DataTypes.NOW,
+    },
+  })
+}
+/**
+ * revert the Experiment Value Migration
+ */
+async function down({ context: queryInterface }) {
+  await queryInterface.dropTable("Value")
```
Comment (niklhut) [inline server/database/migrations/experiment/00_Value.js:40]: Also adjust name here

File: server/database/seeds/experiments/attribute.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import { v4 } from "uuid"
+/**
+ * Seed the Attribute Table with initial data
+ */
+export const up = async ({ context: queryInterface }) => {
+  await queryInterface.bulkInsert("Attribute", [
```
Comment (niklhut) [inline server/database/seeds/experiments/attribute.js:6]: Also adjust the names here

File: server/database/seeds/experiments/attribute.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+    { id: v4(), name: "Attribut-1", createdAt: new Date(), updatedAt: new Date() },
+  ])
+}
+/**
+ * Revert the seeding of the Attribute Table
+ */
+export const down = async ({ contect: queryInterface }) => {
+  await queryInterface.bulkDelete("Attribute", null, {})
```
Comment (niklhut) [inline server/database/seeds/experiments/attribute.js:14]: and here

File: server/database/seeds/experiments/value.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
+import { v4 } from "uuid"
+/**
+ * Seed the Value Table with initial data
+ */
+export const up = async ({ context: queryInterface }) => {
+  await queryInterface.bulkInsert("Value", [
```
Comment (niklhut) [inline server/database/seeds/experiments/value.js:6]: And here

File: server/database/seeds/experiments/value.js [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+    { id: v4(), attributeValue: v4(), name: "Value-1", createdAt: new Date(), updatedAt: new Date() },
+  ])
+}
+/**
+ * Revert the seeding of the Attribute Table
+ */
+export const down = async ({ contect: queryInterface }) => {
+  await queryInterface.bulkDelete("Value", null, {})
```
Comment (niklhut) [inline server/database/seeds/experiments/value.js:14]: And here

**Review by niklhut — CHANGES_REQUESTED** [review]
Looks good, but the table names and file names are incorrect and the migrations and seeds should not be in subfolders because otherwise they are not found.

File: server/database/migrations/00_experiment_attribute_value.js [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+  await queryInterface.createTable("ExperimentAttributeValues", {
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      defaultValue: DataTypes.UUIDV4,
+    },
+    attributeValue: {
+      type: DataTypes.UUIDV4,
+      refereces: {
+        model: ExperimentAttribute,
```
Comment (niklhut) [inline server/database/migrations/00_experiment_attribute_value.js:16]: And put here the table name here in a string so "ExperimentAttributes"

**Review by niklhut — APPROVED** [review]
LGTM

**Pull request merged**